artifacts/BOOTSTRAP_README.md

```diff
@@ -0,0 +1,43 @@
+# Foundation Bootstrap Team
```

Comment:
This document could probably do with a short intro, something like "The following is an archived copy of the bootstrap-stage readme document."

Or maybe even be deleted entirely, as it is stored in the git history? One of the last commits before the CPC was formed could be tagged with something like `bootstrap` to allow the documents at that point in time to be more easily accessible.

Reply:
Updated with a note

Reply:
Also, I think it is good that we keep this as a historical document. Alternatively we should delete the entire "archived" directory. If we want to do that I think it would make sense to make that change in a future PR with that specific intent